src/scripts/h5p-dictation.js

```diff
@@ -737,6 +740,17 @@ class Dictation extends H5P.Question {
      * @returns {string} Description.
      */
     this.getDescription = () => this.params.taskDescription || Dictation.DEFAULT_DESCRIPTION;
+
+    /**
+     * Checks if any sentence's audio playback has started.
+     * @returns {boolean} True if audio playback has started, false otherwise.
+     */
+    this.isAudioPlayBackStarted = () => {
+      return this.sentences?.some((sentence) => {
+        return sentence.getCurrentState().buttonPlayNormal?.audio?.currentTime > 0 
```

Comment:
It's not necessary to go via `getCurrentState` to retrieve the buttons. They are exposed in the `Sentence` class.

I'd prefer to either access the buttons directly like `sentence.buttonPlayNormal` / `sentence.buttonPlaySlow` and to determine their state or to rather introduce an `isSomeButtonPlaying : boolean` function. Either way an `isPlaying : boolean` function in `Button` should be used.